projects/angular/src/data/datagrid/all.spec.ts

```diff
@@ -81,6 +82,7 @@ describe('Datagrid', function () {
     DatagridCellSpecs();
     DatagridFilterSpecs();
     DatagridColumnSpecs();
+    DatagridDetailSpecs();
```

Comment:
I think the tests should be included and fixed separately to avoid including unrelated test fixes in the same commit as this bug fix. That can be confusing to readers. I was confused at first. I have opened #1451 for this.

Reply:
Oh yes, I meant to discuss this with you. The older tests were commented out and were failing.
Makes sense to move it to a new PR. I have approved https://github.com/vmware-clarity/ng-clarity/pull/1451
Please check.

Reply:
Thanks, my PR is merged now.